background/messages/createEntry.ts

```diff
@@ -15,7 +15,7 @@ const handler: PlasmoMessaging.MessageHandler<
   CreateEntryRequestBody,
   CreateEntryResponseBody
 > = async (req, res) => {
-  if (req.body && (await getClipboardMonitorIsEnabled())) {
+  if (req.body && req.body.content && (await getClipboardMonitorIsEnabled())) {
```

Comment:
Move the check to line 25. This is because the popup uses `ClipboardSnapshot` to display what's currently copied in the clipboard. We can't just use the most recent entry because copying an item by clicking on it in the popup doesn't update the list.

What can happen now is when a user copies an image `ClipboardSnapshot` won't get updated (line 24). When the user opens up the popup it'll show an item is copied when it actually isn't.